src/elfpy/utils/parse_config.py

```diff
@@ -60,7 +60,6 @@ def parse_simulation_config(config_dict):
         amm=AMMConfig(**config_dict["amm"]),
         simulator=SimulatorConfig(**config_dict["simulator"]),
     )
-    simulation_config.simulator.logging_level = text_to_logging_level(simulation_config.simulator.logging_level)
```

Comment:
The default value for the SimulatorConfig dataclass logging level is a string, but the logging level needs to be an int. the `text_to_logging_level` function makes this conversion happen. I am probably missing something, but why are you able to remove this conversion here and elsewhere?

Reply:
I can understand the confusion. The impetus of the change was that the previous code was assigning a string to the config and then assigned an int to the config, which was problematic from a typing perspective (and pyright complained about it). What I've done instead of this conversion is keep the configuration as a string always but pass the log level (string) into `test_to_logging_level` when I pass the log level into the logger. I'm going to abstract this logic to avoid the linter's code duplication complaints, so I will write a function that takes care of this.